src/prognostic_equations/edmfx_closures.jl

```diff
@@ -184,7 +184,7 @@ function pi_groups_entr_detr(
         # TODO - Temporary: Switch to Π groups after simple tests are done
         # (kinematic, bubble, Bomex)
         # and/or we can calibrate things in ClimaAtmos
-        entr = max(0, 1e-1 * ᶜwʲ / ᶜz)
+        entr = max(0, 1 * ᶜwʲ / ᶜz)
```

Comment:
Another hack for bomex. We don't have to change it though, maybe we can just get this PR in and work on making bomex look better after adding other tendencies?